gating/thaw/fix_endpoints.yml

```diff
@@ -0,0 +1,15 @@
+- name: Fix keystone endpoints
+  hosts: localhost
+  tasks:
+    - name: Start galera container
+      shell: |
+        lxc-start -d -n {{ groups['galera_all'][0] }}
+    - name: Wait for MySQL to become available
+      wait_for:
+        port: 3306
+        delay: 30
+      delegate_to: "{{ groups['galera_all'][0] }}"
+    - name: Update endpoint table
+      command: |
+        mysql keystone -e "UPDATE endpoint SET url = REPLACE(url, '{{ orig_ip }}', '{{ ansible_default_ipv4.address }}')";
```

Comment:
is there a reason we reach into the DB instead of using keystone-manage or the openstack command?